specs/commandline/plugin.md

```diff
@@ -36,15 +36,55 @@ Aliases:
   list, ls
 ```
 
+### notation plugin install
+
+```text
+Installs a plugin
+
+Usage:
+  notation plugin install [--name <plugin name>] [flags] <plugin package> 
+
+Flags:
+  -n, --name           string   name of the plugin
```

Comment:
We may don't need the name of the plugin because the name is the required field of `get-plugin-metadata` plugin command. [plugin spec](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md#plugin-metadata)
Also, the name of the directory for storing the plugin should be the same with the name returned by `get-plugin-metadata` command, so the user cannot change the name and there is no need to set it.

Reply:
The `--name` flag is optional. It is possible that the plugin in the package is not the intended plugin. Having `--name` adds an additional assurance.

See also https://github.com/notaryproject/notation/pull/555#discussion_r1108298855

Reply:
The [plugin extensibility](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md#plugin-lifecycle-management) spec lays out a naming standard for the plugin folder and the binary, will notation load the plugin if it doesn't follow the naming convention?

Reply:
Resolving based on the Outdated label, but please feel free to start a new thread or reopen if desired. :)